app/src/main/java/com/globant/practice/presentation/presenter/SubscriberDetailsPresenter.java

```diff
@@ -124,10 +120,8 @@ public SubscriberDetailsPresenter(FetchSubscriberProfileInteractor profileIntera
      */
     public void fetchSubscriberDetails(String login) {
         subscriberDetailsState.setLogin(login);
-        if (subscriberDetailsState.isLoading() && subscriberProfile != null && subscriberRepositories != null) {
+        if (subscriberDetailsState.getProfile() != null && subscriberDetailsState.getSubscriberRepositories() != null) {
```

Comment:
What if Profile and Repositories are null, the request is being performed but there is some delay in the Internet connection, the user changes the screen orientation, is a second request executed?